Client/src/Components/Graph/Graph.tsx

```diff
@@ -34,16 +34,86 @@ export default function Graph(props: IProps) {
 
   const [isXLog, setXToggle] = useState(false)
   const [isYLog, setYToggle] = useState(false)
+  const [minX, setMinX] = useState(9999)
```

Comment:
use one object to define all this, also, are Min/Max values really needed? whats the problem with a user setting the bound too far, also here is a nice chart from another lib that has that feature embedded: https://www.amcharts.com/demos/dumbbell-plot/

Reply:
I guess overall, we can just give the tools to the user, and let them do what they want. But if the boundaries are there, i still want them respected. So no zooming out further than the bounds or dragging it out of bounds. The only different with your example is the lack of the "drag" feature directly on the graph itself